### PR TITLE
KEYCLOAK-16294 Updating IBM Power Systems terminology

### PR DESCRIPTION
**Pull request by rahsin11**
Updates for Power architecture has been added, If anything more is required, please let me know in the review

JIRA: https://issues.redhat.com/browse/KEYCLOAK-16294

### CODE REVIEW

**Comment by rahsin11** [comment]
@andymunro Can you please review these changes for Power arch.

File: release_notes/topics/product/7_4_final.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Red Hat is proud to announce the release of version 7.4 of {project_name} (RH-SSO). RH-SSO is based on the Keycloak project, and enables you to secure your web applications by providing Web SSO capabilities based on popular standards such as OpenID Connect, OAuth 2.0, and SAML 2.0. The RH-SSO server acts as an OpenID Connect or SAML-based identity provider (IdP), allowing your enterprise user directory or third-party IdP to secure your applications via standards-based security tokens.
 
 [NOTE]
-{project_name} for IBM Z is supported only in the OpenShift environment. Bare metal installations on IBM Z are not supported.
+{project_name} for IBM Z/IBM Power is supported only in the OpenShift environment. Bare metal installations on IBM Z/IBM Power are not supported.
```
Comment (SNiemann15) [inline release_notes/topics/product/7_4_final.adoc]: @rahsin11 Please use correct brand names:  IBM Z and IBM Power Systems

{project_name} for IBM Z and IBM Power Systems is supported only in the OpenShift environment. Bare metal installations on IBM Z and IBM Power Systems are not supported.

File: topics/templates/document-attributes-product.adoc [hunk 2]
```diff
@@ -70,7 +70,7 @@
 :openshift_openjdk_project_templates_version: sso74
 :openshift_openj9_name: Red Hat Single Sign-On for OpenShift on Eclipse OpenJ9
 :openshift_openj9_link: {project_doc_base_url}/red_hat_single_sign-on_for_openshift_on_openj9/
-:openshift_openj9_platforms: IBM Z
+:openshift_openj9_platforms: IBM Z/ IBM Power
```
Comment (SNiemann15) [inline topics/templates/document-attributes-product.adoc]: IBM Z/IBM Power Systems

**Review by andymunro — DISMISSED** [review]
This looks good. Thanks

**Comment by rahsin11** [comment]
I made the changes as per @SNiemann15 review and squashed the commits. @andymunro can it be merged now.

**Comment by andymunro** [comment]
@pskopek Can this be merged now. It is a very small change.

**Comment by andymunro** [comment]
@abstractj or @hmlnarik can you approve this and merge it?

**Review by hmlnarik — CHANGES_REQUESTED** [review]
This PR is missing JIRA. @rahsin11 / @andymunro Please file a JIRA and update the PR.

**Comment by rahsin11** [comment]
> This PR is missing JIRA. @rahsin11 / @andymunro Please file a JIRA and update the PR.

**Comment by rahsin11** [comment]
I have updated the JIRA in the PR @hmlnarik

**Pull request merged**